datadog_checks_base/datadog_checks/base/checks/win/wmi/__init__.py

```diff
@@ -2,13 +2,21 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 from collections import namedtuple
+from typing import Any, Dict, Iterable, List, Optional, Tuple, Union
 
 from six import iteritems
 
+from datadog_checks.base import ConfigurationError
+from datadog_checks.base.errors import CheckException
+
 from ... import AgentCheck
 from .sampler import WMISampler
 
 WMIMetric = namedtuple('WMIMetric', ['name', 'value', 'tags'])
+WMIProperties = Tuple[Dict[str, Tuple[str, str]], Iterable[str]]
+TagQuery = List[str]
+WMIObject = Dict
+WMIFilter = Union[str, List[str]]
```

Comment:
(Nit) Should we move these to a `types.py` module? In the future they could be reused elsewhere (eg in `sampler.py`, or in `WMICheck` for type checking, etc). I consider this a nit because we can also make that refactor when it's actually needed, but at the same time it's a small beneficial step that we might forget to do later…